source/source_cell/unitcell.cpp

```diff
@@ -329,11 +345,13 @@ void UnitCell::setup_cell(const std::string& fn, std::ofstream& log)
     //===================================
     this->set_iat2itia();
 
+    GlobalC::sep_cell.set_omega(this->omega, this->tpiba2);
```

Comment:
maybe you can move GlobalC::sep_cell.set_omega to esolver instead of inside unitcell, then you can avoid using sep_cell as a GlobalC